umd-fcrepo-event-router/src/main/cfg/edu.umd.lib.fcrepo.camel.router.cfg

```diff
@@ -19,3 +19,6 @@ batch.jms.priority=4
 
 # Skip indexing of "/pcdm" node on batch mode
 batch.skip.pcdm.container=true
```

Comment:
That is true. I was leaving the `batch.skip.pcdm.container` in there for backwards compatibility. I could remove it, though.